src/99-appendices/11-qmri.md

```diff
@@ -0,0 +1,547 @@
+magnetization prepared two gradient echoes (MP2RAGE). Such generic acquisitions
+can serve as a basis to derive various qMRI applications by changes to the acquisition
+sequence (e.g. readout) type or by varying additional scan parameters.
+
+If such an inheritance relationship is applicable between an already existing file
+collection and a new qMRI application to be included in the specification, the inheritor
+qMRI method is listed in the table below instead of introducing a new file collection suffix.
+This approach aims at:
+
+* preventing the list of available suffixes from over-proliferation,
+* providing qMRI-focused BIDS applications with a set of meta-data driven rules to infer 
+possible fitting options,
+* keeping an inheritance track of the qMRI methods described within the specification.
+
+| **File-collection suffix** | **If REQUIRED metadata == Value** | **OPTIONAL metadata [`entity`/`fixed`]**   | **Derived application name (NOT a suffix)** |
+|----------------------------|-----------------------------------|--------------------------------------------|---------------------------------------------|
+| VFA                        | `PulseSequenceType` == `SPGR`     |                                            | DESPOT1                                     |
+| VFA                        | `PulseSequenceType` == `SSFP`     | `SpoilingRFPhaseIncrement` [`fixed`]       | DESPOT2                                     |
+| MP2RAGE                    |                                   | `EchoTime` [`echo`]                        | MP2RAGE-ME                                  |
+| MPM                        |                                   | `EchoTime` [`echo`]                        | MPM-ME                                      |
+
+<sup>*</sup> [`entity`/`fixed`] denotes whether the OPTIONAL metadata that forms a new
+flavor of qMRI applicaiton for the respective suffix varies across files of a file colleciton
+(which calls for using a linking entity) or fixed. If former is the case, the entity is to be
+added to the files in that file collection. Note that this addition MUST be allowed by the
+priority levels given for that suffix in the [`entity table`](../99-appendices/04-entity-table.md). If latter (`fixed`) is the case,
+filenames will remain the same; however, the optional metadata (third column) may
+define the flavor of the application (fourth column) along with the conditional value of a
+required metadata field (second column).
+
+A derived qMRI application becomes avaiable if all the optional metadata fields
+listed for the respective file collection suffix are provided for the data. In addition,
+conditional rules based on the value of a given required metada field can be set
+for the description of a derived qMRI application. Note that the value of this
+required metadata is fixed across constituent images of a file collection and defined
+in the _Method-specific priority levels for qMRI file collections_ section above.
```

Comment:
please use a link to that section here, instead of italics

Reply:
0630127aa5b55211a64bad4a63cca0c127ad8c67